packages/block-editor/src/components/block-removal-warning-modal/index.js

```diff
@@ -41,6 +41,18 @@ export function BlockRemovalWarningModal( { rules } ) {
 		return;
 	}
 
```

Comment:
could just do a ternary here, but these seems easier to read with these long vars, and also easier to add more message types later this way.